raft/raft.go

```diff
@@ -1461,17 +1469,14 @@ func (r *raft) applyConfChange(cc pb.ConfChangeV2) pb.ConfState {
 
 	r.prs.Config = cfg
 	r.prs.Progress = prs
+	return r.applyConfChangePost()
+}
 
+// applyConfChangePost is called from restore and applyConfChange to update any
```

Comment:
"from newRaft, restore, and applyConfChange"

Also, I don't think you meant to say "update any updates". Did you mean process?

Reply:
I'm also not in love with this name. It doesn't really tell me anything. It would be more effective if it mentioned what it does.

Reply:
There was lots to not love about this. Now this method is in charge of actually setting the config. This turned out nicely I think, PTAL